bookkeeper-server/src/test/java/org/apache/bookkeeper/client/LedgerRecoveryTest.java

```diff
@@ -480,14 +481,15 @@ public void addComplete(int rc, LedgerHandle lh, long entryId, Object ctx) {
 
         final CountDownLatch recoverLatch = new CountDownLatch(1);
         final AtomicBoolean success = new AtomicBoolean(false);
-        LedgerRecoveryOp recoveryOp = new LedgerRecoveryOp(recoverLh,
+        LedgerRecoveryOp recoveryOp = new LedgerRecoveryOp(
+                recoverLh, bkc.getClientCtx(),
                 new BookkeeperInternalCallbacks.GenericCallback<Void>() {
-            @Override
-            public void operationComplete(int rc, Void result) {
-                success.set(BKException.Code.OK == rc);
-                recoverLatch.countDown();
-            }
-        }).parallelRead(true).readBatchSize(newConf.getRecoveryReadBatchSize());
+                    @Override
+                    public void operationComplete(int rc, Void result) {
```

Comment:
I think this changes the testing behavior. the original testing behavior is `parallelRead` only enabled on `LedgerRecoveryOp`. but this PRs changes the behavior to apply to all behaviors on `newBk`, hences all recovery ops will inherit the settings. I am not sure is it okay or not, but I am not very conformatable about such behavior change in a refactor PR.

Reply:
I suspect that it's not actually important for the test, but I've changed it to the exact original behaviour in any case.